### PR TITLE
Use zone from the test context for static IPs.

### PR DESCRIPTION
**Pull request by spxtr**

### CODE REVIEW

File: test/e2e/google_compute.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ func createGCEStaticIP(name string) (string, error) {
 	for attempts := 0; attempts < 4; attempts++ {
 		outputBytes, err = exec.Command("gcloud", "compute", "addresses", "create",
 			name, "--project", framework.TestContext.CloudConfig.ProjectID,
-			"--region", "us-central1", "-q").CombinedOutput()
+			"--region", framework.TestContext.CloudConfig.Zone, "-q").CombinedOutput()
```
Comment (maisem) [inline test/e2e/google_compute.go]: You will have to use gce.GetGCERegion(framework.TestContext.CloudConfig.Zone).

File: test/e2e/google_compute.go [hunk 2]
```diff
@@ -79,7 +79,7 @@ func deleteGCEStaticIP(name string) error {
 
 	outputBytes, err := exec.Command("gcloud", "compute", "addresses", "delete",
 		name, "--project", framework.TestContext.CloudConfig.ProjectID,
-		"--region", "us-central1", "-q").CombinedOutput()
+		"--region", framework.TestContext.CloudConfig.Zone, "-q").CombinedOutput()
```
Comment (maisem) [inline test/e2e/google_compute.go]: Same here.

File: test/e2e/google_compute.go (hunk 1 above)
Comment (spxtr) [inline test/e2e/google_compute.go]: Done.

File: test/e2e/google_compute.go [hunk 3]
```diff
@@ -44,7 +45,7 @@ func createGCEStaticIP(name string) (string, error) {
 	for attempts := 0; attempts < 4; attempts++ {
```
Comment (maisem) [inline test/e2e/google_compute.go]: Mind adding the region in the log?

Comment (spxtr) [inline test/e2e/google_compute.go]: Done.

**Comment by maisem** [comment]
Thanks for helping me with this!

**Comment by jlowdermilk** [comment]
@spxtr can you cherrypick this? Jobs that run pinned to the 1.2 branch in non-us-central1 zones will be broken until this is on the branch.

**Pull request merged**